### ISSUE TITLE
rultor not timing out

### ISSUE THREAD

**Issue by AbdealiJK** [yegor256/rultor#1042]
Hey,

I was testing some functionality on rultor and forgot to add the `-y` flag in apt-get. Because of this, apt-get inside rultor asked for a prompt and got stuck.
Logs: http://www.rultor.com/t/7510-192639053
PR: https://github.com/AbdealiJK/pycolorname/pull/15#issuecomment-192639066

I tried aborting it with the rultor commands, but it didnt seem to work. I then manually pushed the changs to master in the hopes that when the PR closes rultor would stop - but it doesnt seem to have done that either.

I think the container is still there on the rultor (As rultor hasnt reported back saying anything like "Timed out - here's your log")

Could you check up on this ? I think a max time limit of 15-30 mins should be imposed.

**Comment by original-brownbear** [comment]
@AbdealiJK there actually is a timeout of 2h imposed.
But I agree, we should have a timeout in case of no output for 15 minutes.
Another issue here is that the stop command appears broken, we already have a ticket for this here #864.

Whoever handles this, please implement a **15 minute timeout** killing a build **when not output is received for 15 minutes**. Travis in fact does the exact same as described [here](https://docs.travis-ci.com/user/common-build-problems/#My-builds-are-timing-out).

**Comment by original-brownbear** [comment]
@alex-palevsky this is a bug.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is a bug.

@original-brownbear I added `bug` tag to this ticket

**Comment by alex-palevsky** [comment]
@abdealijk attached this issue to milestone "2.0" (let me know if this is wrong)

**Comment by original-brownbear** [comment]
@alex-palevsky this depends on #1055.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this depends on #1055.

@original-brownbear agreed, we'll wait for #1055

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear right, I added "postponed" label

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear no problem, I will try to find somebody else

**Comment by alex-palevsky** [comment]
@abdealijk we waited for #1055, it is closed already
